pkg/provider/senhasegura/dsm/dsm.go

```diff
@@ -190,7 +190,7 @@ func (dsm *DSM) FetchSecrets() (respObj IsoDappResponse, err error) {
 		return respObj, errInvalidHTTPCode
 	}
 
-	respData, err := ioutil.ReadAll(resp.Body)
+	respData, err := io.ReadAll(resp.Body)
```

Comment:
`pkg/provider/senhasegura/auth/iso.go` and `pkg/provider/senhasegura/dsm/dsm.go` only touched to make `make lint` pass.

Reply:
no longer doing this after changing the linter version.